src/main.test.js

```diff
@@ -169,3 +169,25 @@ test("Wrong Syntax for default value,missing value", () => {
     message: "EnvSpecSyntaxError: Expected default value after ="
   });
 });
+
+test("Test for serialization of data", () => {
+  const myForm = document.createElement("form");
+  const aLabel = document.createElement("label");
+  const anInput = document.createElement("input");
```

Comment:
Add a value please to this `input` and test that it is being serialized successfully.